tensorrt_llm/_torch/pyexecutor/mamba_cache_manager.py

```
@@ -670,6 +774,60 @@ def _prepare_mamba_cache_blocks(self, request_ids: List[int]):
+        d = conv.shape[2] // 3
+        committed = conv.shape[3] - 1  # W - 1
+        cs = conv.index_select(1, idx)
+        for cache, section in (
+            (self.mamba_cache.kda_conv_q, cs[:, :, :d]),
+            (self.mamba_cache.kda_conv_k, cs[:, :, d:2 * d]),
+            (self.mamba_cache.kda_conv_v, cs[:, :, 2 * d:]),
+        ):
+            # cache: [L, slots, D, committed + num_spec]; zero the draft
+            # tail columns and seed the committed window in one copy.
+            seeded = torch.zeros(
+                (cache.shape[0], idx.numel()) + cache.shape[2:],
```

Ruff (0.16.1)

[warning] Consider `(cache.shape[0], idx.numel(), *cache.shape[2:])` instead of concatenation

Replace with `(cache.shape[0], idx.numel(), *cache.shape[2:])`

(RUF005)